### PR TITLE
k8s: Update to 1.18

### PR DESCRIPTION
**Pull request by surajssd**
- This PR updates the cluster to use 1.18.0 images
- For all the control plane components (apiserver, scheduler, controller-manager, kube-proxy) we have been using hyperkube images. This PR changes the images to use the component specific images.

---

Something that is not done in this PR is changing the images of kubelet. Kubelet is still provisioned using hyperkube. Upstream will stop shipping any image for kubelet. We need to build and maintain images for kubelet or we can use the images that are built for typhoon: `quay.io/poseidon/kubelet:v1.17.4`.

If you folks are ok with using that image then I can change that in this PR as well?

### CODE REVIEW

File: assets/lokomotive-kubernetes/aws/flatcar-linux/kubernetes/cl/controller.yaml.tmpl [hunk 1]
```diff
@@ -121,7 +121,7 @@ storage:
       mode: 0644
       contents:
         inline: |
-          KUBELET_IMAGE_URL=docker://k8s.gcr.io/hyperkube
```
Comment (invidian) [inline assets/lokomotive-kubernetes/aws/flatcar-linux/kubernetes/cl/controller.yaml.tmpl]: Why is this needed? The hyperkube images works just fine for now. I think we need a bit more thought in how to handle their deprecation.

Comment (surajssd) [reply assets/lokomotive-kubernetes/aws/flatcar-linux/kubernetes/cl/controller.yaml.tmpl]: I have asked question about it in the description.

**Comment by surajssd** [comment]
@invidian issue for kubelet images: https://github.com/kinvolk/lokomotive/issues/221
